### ISSUE TITLE
[Bug]: Some elements have scaling issues with increased custom font size in the new Material Design Style

### ISSUE THREAD

**Issue by stilesdev** [mallowigi/material-theme-issues#121]
### Describe the bug

Certain elements do not scale properly with the rest of the UI when a custom font is configured with a larger font size. In my case, I am using Cantarell size 18 to scale the UI up on a 4k monitor.

The new file/folder text input is unreadable:
![image](https://user-images.githubusercontent.com/1377201/161775167-5121feee-b9e4-4fec-8841-0f8b7ac32628.png)

A few other minor issues, but still usable:
Number input arrows get cut off, dropdown arrows are just barely touching the border:
![image](https://user-images.githubusercontent.com/1377201/161776098-ec7a9570-c921-498e-a4f0-f3b63ee1a7a1.png)

Search Everywhere/Run Anything text inputs are slightly cut off but still readable:
![image](https://user-images.githubusercontent.com/1377201/161780802-d27ce829-45eb-4791-9d01-36a8161e5731.png)

### To Reproduce

1. Install Material Theme UI
2. Set a larger custom font size under Settings - Appearance & Behavior - Appearance
![image](https://user-images.githubusercontent.com/1377201/161774461-b0bab9ad-57ec-4a0a-86b5-312de56341d4.png)
3. Restart the IDE (you are not prompted to restart when changing this setting, but it doesn't seem to fully take effect until restarting the IDE)
4. Review the affected elements listed above

I typically have Compact Fields enabled. I tried with it disabled, and it is less cut off, but not completely fixed.

I also tried with and without the custom font settings in the various tabs of the Material Theme UI settings, and it didn't seem to make a difference.

This all started happening when the new Material Design style was added. Reverting to the old Material Design style fixes all of these issues, but I would prefer to use the new style.
![image](https://user-images.githubusercontent.com/1377201/161784206-f3baebcd-1fbb-4e5c-87e9-7829fba4158c.png)

### Expected behavior

These last few elements should scale with the rest of the UI when a larger custom font size is selected.

### Material Theme Version

v6.16.2

### IDE

PHPStorm

### System Info

```shell
PhpStorm 2021.3.3
Build #PS-213.7172.28, built on March 17, 2022
Licensed to Matthew Stiles
Subscription is active until October 23, 2022.
Runtime version: 11.0.14.1+1-b1751.46 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.17.1-arch1-1
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 20
Registry:
    run.processes.with.pty=TRUE
    ide.balloon.shadow.size=0

Non-Bundled Plugins:
    ideanginx9 (0.1.9)
    String Manipulation (9.4.0)
    zielu.gittoolbox (212.8.8)
    izhangzhihao.rainbow.brackets (6.21)
    com.dmarcotte.handlebars (213.5744.190)
    com.chrisrm.idea.MaterialThemeUI (6.16.2)
    com.mallowigi (54.1.0)
    ru.adelf.idea.dotenv (2022.1)

Current Desktop: i3
```

**Comment by mallowigi** [comment]
I can reproduce the  "new popup" issue with the compact fields enabled. I'll fix this.

**Comment by MetaHegemon** [comment]
MacBook pro


![image](https://user-images.githubusercontent.com/7349599/162563045-eae9fda1-0329-454e-8478-a1ac67ceb436.png)

![image](https://user-images.githubusercontent.com/7349599/162563116-862cad1f-7ff3-4713-b41a-d6ebc24016c0.png)

**Comment by mallowigi** [comment]
Fixed in 6.16.2

Also, I've stopped updating 2021 releases since 2022.1 is around the corner.
